sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -783,6 +784,28 @@ def odfv1(feature_df: pd.DataFrame) -> pd.DataFrame:
     assert (
         updated_feature_view.created_timestamp is not None
         and updated_feature_view.created_timestamp == feature_view.created_timestamp
+        and len(updated_feature_view.materialization_intervals) == 0
+    )
+
+    # Simulate materialization
```

Comment:
You should put this before `test_registry.apply_feature_view(updated_fv1, project)` to make sure that an apply operation doesn't mess with materialization intervals.